nx-dev/nx-dev/redirect-rules.config.js

```diff
@@ -366,17 +366,17 @@ const reactTutorialRedirects = oldReactTutorialPaths.reduce((acc, path) => {
   return acc;
 }, {});
 const oldNodeTutorialPaths = [
-  '/node-tutorial/01-create-application',
-  '/node-tutorial/02-display-todos',
-  '/node-tutorial/03-share-code',
-  '/node-tutorial/04-create-libs',
-  '/node-tutorial/05-dep-graph',
-  '/node-tutorial/06-computation-caching',
-  '/node-tutorial/07-test-affected-projects',
-  '/node-tutorial/4-workspace-optimization',
-  '/node-tutorial/08-summary',
+  '/node-server-tutorial/01-create-application',
```

Comment:
The `oldNodeTutorialPaths` should be left as is since it is the previous version (before the Node effort).

Reply:
okay I'll revert these ones